### ISSUE TITLE
Github actions release doesn't work

### ISSUE THREAD

**Issue by japgolly** [suzaku-io/boopickle#128]
https://github.com/suzaku-io/boopickle/runs/745558731?check_suite_focus=true

```
[error] Expected ';'
[error] Not a valid command: ci-release
[error] Not a valid project ID: ci-release
[error] Expected ':'
[error] Not a valid key: ci-release (similar: clean, credentials)
[error] ci-release
[error]           ^
##[error]Process completed with exit code 1.
```

/cc @cquiroz Hey man. Are you able to get this working?

**Comment by cquiroz** [comment]
Let's see if #129 is enough to fix it.

**Comment by japgolly** [comment]
Closed via 5fb3563cb3bf44a0b1c1260b26ed13dd81049a17
